fastapi_rss/models/feed.py

```diff
@@ -107,10 +107,15 @@ def generate_tree(root: etree.Element, dict_: dict):
                     break
             else:
                 RSSFeed._generate_tree_default(root, key, value)
+                if key == "docs" and 'http://www.w3.org/2005/Atom' in nsmap.values():
+                    atom_link = etree.SubElement(root, '{http://www.w3.org/2005/Atom}link', nsmap=nsmap)
+                    atom_link.set('href', value)
+                    atom_link.set('rel', 'self')
+                    atom_link.set('type', 'application/rss+xml')
 
     def tostring(self, nsmap: Optional[Dict[str, str]] = None):
-        nsmap = nsmap or {}
+        nsmap = nsmap or {'atom': 'http://www.w3.org/2005/Atom'}
         rss = etree.Element('rss', version='2.0', nsmap=nsmap)
         channel = etree.SubElement(rss, 'channel')
-        RSSFeed.generate_tree(channel, self.dict())
-        return etree.tostring(rss, pretty_print=True, xml_declaration=True)
+        RSSFeed.generate_tree(channel, self.dict(), nsmap)
+        return etree.tostring(rss, pretty_print=True, xml_declaration=True, encoding='UTF-8')
```

Comment:
I should have originally encoded the string as utf-8, I agree, but the original RSS specification didn't mention a specific encoding to use (to be fair, at the time of the spec for RSS 2.0, UTF-8 wasn't widely supported yet).

Encoding as UTF-8 can be the default, but it should be able to be opted out if need be, in order to keep the library consistent with its previous version.

Furthermore, such a change will warrant a major version bump. Please upgrade the version in pyproject.toml to reflect that.